src/ontology/analysis/analyze_mondo_stats/README.md

```diff
@@ -0,0 +1,333 @@
+
+NOTE: A list of all NCIT terms in the neoplasm branch is needed in order to filter all Mondo xrefs to NCIT where these NCIT terms are in the neoplasm branch since Mondo contains NCIT xrefs to terms both in the neoplasm branch and other NCIT branches.
+
+
+- Run from `mondo/src/ontology/analysis/analyze_mondo_stats` as: 
+```
+robot reason -i ../../mondo-edit.obo query --use-graphs true -q sparql/get_all_human_disease_mondo_classes_with_xref.ru data/output/all_human_disease_mondo_class_xref_count.tsv
+```
+
+
+#### Sources of Interest:
+- DOID
+- NCIT (neoplasm branch) --> special processing needed
+- OMIM
+- Orphanet - all terms and also only orphanet disorder and disease subtypes
+- ICD10CM
+- UMLS
+
+
+#### Orphanet Disorder and Disease Subtypes
+- Run from `mondo/src/ontology/analysis/analyze_mondo_stats` as:
+```
+robot reason -i ../../mondo-edit.obo query --use-graphs true -q sparql/get_all_human_disease_mondo_classes_with_xref_ORDO-disorder-subtypes.ru data/output/all_human_disease_mondo_class_xref_ORDO-disorder-subtypes_count.tsv
+```
+
+#### Alignment to NCIT
+##### Get all classes from NCIT that are in the neoplasm branch
+- Run from `mondo/src/ontology/analysis/analyze_mondo_stats` as:
+```
+robot reason -i ./data/component-download-ncit.owl.owl query --use-graphs true -q sparql/ncit_neoplasm_children.ru ./data/input/ncit_neoplasm_children.tsv
+```
+The file `component-download-ncit.owl.owl` was copied from the mondo-ingest repo from `mondo-ingest/src/ontology/tmp`.
+
+- Query Mondo for a _list_ of all Mondo classes that are in the human disease branch, i.e. children of MONDO:0700096 'human disease' and have an xref to NCIT. Run from `mondo/src/ontology/analysis/analyze_mondo_stats` as: 
+```
+robot reason -i ../../mondo-edit.obo query --use-graphs true -q sparql/get_mondo_class_list_with_ncit_xref.ru data/input/all_mondo_classes_with_ncit_xref.tsv
+```
+
+- Check for overlapping NCIT CURIEs in each file by running these commands from the Terminal:
+```
+$ awk -F'\t' 'NR==FNR {xref[$1]; next} $3 in xref' data/ncit_neoplasm_children.tsv data/input/all_mondo_classes_with_ncit_xref.tsv > data/all_mondo_classes_with_ncit_xref_neoplasm-branch.tsv
+
+$ wc data/all_mondo_classes_with_ncit_xref_neoplasm-branch.tsv
+    3804   20177  229560 data/all_mondo_classes_with_ncit_xref_neoplasm-branch.tsv
+```
+
+#### Results for Alignment with the source - Total number of diseases in sources
+DOID	11390
+ICD10CM	9142
+NCIT	3804
+OMIM	9333
+Orphanet	11035 (all terms)
+Orphanet    7218 (ordo_disorder and ordo_subtype_of_a_disorder subsets)
+UMLS	20738
+
+
+### Alignment with the source - Number of terms from the sources that are in Mondo (as Mondo:equivalent)
+To generate these counts, follow the steps above for "Alignment with the source - Total number of diseases in sources", but create new queries where the xref must also have a source annotation of `MONDO:equivalentTo`.
```

Comment:
Create a fully executable pipeline in the Makefile, like `metrics`, so you can run

```
sh run.sh make metrics
```

and get everything.

Reply:
Again, creating a a repeatable workflow is out of scope for this work. It was only to generate stats for the grant and  document the steps as Sabrina requested.